tools/hooks/30-json.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+echo "Executing 'json' formatter..."
+
+TARGET_FILES=`$TRACKED_FILES | grep -P '\.json'`
+
+pushd $PROJECT_ROOT > /dev/null
+    for FILE in ${TARGET_FILES}; do
+        python -mjson.tool $FILE 2>&1 /dev/null
+        if [ $? -ne 0 ] ; then
```

Comment:
This hook should reformat json to pretty printed, key sorted form. This hook will not error on valid json, even if it is poorly formatted.

Instead, it should try to reformat, similar to black, and let the user know they need to commit again.

In other words, the functionality should be the same as `format_json.py`.